pkg/core/health/check.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright The Karpor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package health
+
+import (
+	"github.com/KusionStack/karpor/pkg/infra/search/storage"
+	"github.com/go-chi/chi/v5"
+	"net/http"
+)
+
+type StorageImpl struct {
+	storage.ResourceStorage
+	storage.SearchStorage
+	storage.ResourceGroupRuleStorage
+}
```

Comment:
The `Storage` structure has implemented the `Storage` interface. No `StorageImpl` is required.
`Storage` structure: pkg/infra/search/storage/elasticsearch/client.go